_posts/2020-03-15-learning-to-share-accomplishments-2.md

```diff
@@ -0,0 +1,214 @@
+</figure>
+This is the second part of a two-part post. The first part is more on how to share your career accomplishments. You 
+may want to read it first. [Part 1 is here.](../learning-to-share-accomplishments-1/)
+
+For part 2 of this post, I want to actually talk about those accomplishments. I think it's helpful for me to get out 
+there and while it may not be as interesting or relevant for you, maybe in looking it over you can see some ways you 
+can talk about your own accomplishments as well. 
+
+## What I Personally Have Worked on in My Past Jobs
+
+I want to spend some time looking back at my work. Like I said above, I've apparently had some times I've been terrible 
+at talking about this in job interviews, so I wanted a list. I kind of want to use it as a thought exercise for me, 
+but maybe also to help sell myself to anyone who might be reading this post. I don't intend this to be literally 
+everything I've worked on, but perhaps more of the highlights from different points in my career. And who knows, maybe 
+a future employer might see this and think I really would be an awesome addition to their team.
+
+One other thing: I have literally had job interviews where, despite all of my developer experience, have looked at me 
+and said "Looks like you do a lot of teaching stuff. Maybe you should be a teacher and not a developer." I've found 
+it puts me in an incredibly awkward place to have to defend why I am applying for a developer job and these people 
+don't even see me in that light. I'm not sure if that's, again, me not "selling myself" well or if that's me not 
+writing out my resume well enough, or just things like conferences are more visible than projects I've worked on (which 
+are usually NDAs or not publicly available). So I hope this post helps with that as well.
+
+### Best Team: Data Pipeline Team
+
+This team was created to replace a $1M/year third-party product with an in-house solution. The third-party product was 
+expensive, a pain to set up, didn't scale well, and the company only used about 10% of it. Ours was built for this 
+specific solution, scaled with the click of a few buttons, and cost closer to $1000/month (a VAST savings). 
+
+**What did I Do?**
+
+* I was the generalist on this team. I felt like I was the glue that when given a random problem to solve, thing to 
+  research, or item to build, I just did it. This work often helped fix some problems that would have delayed our 
+  release or fix an issue that would have popped up later on. 
+* I figured out the best way to do an AWS-based service queue system for our monitoring microservice. I did the 
+  research by looking at all of AWS's types of queueing systems and their costs of running. I later implemented a basic 
+  prototype, and then wrote the final code to convert our monitoring service to push to it.
+* We had a broken build for our client executable that wasn't building separate versions correctly. I wrote some shell 
```

Comment:
I'm curious (personally, not for the post) what gave you the idea to do this, what kept it from happening before you did it, etc.

Reply:
To do which part?